slave/base/bin/perun

```diff
@@ -131,6 +131,17 @@ function log_msg_without_exit {
 	fi
 }
 
+### Log message to stderr and exit with 0 return code
+function log_warn_to_err_exit {
+	TEXT="$1"
+	TIME=`date "+%H:%M:%S"`
+
+	MSG="Info: ${TEXT}"
+	echo "${TIME} ${MSG}" >&2
+	logger -t "${NAME}" -p daemon.error "${SERVICE}: ${MSG}" &>/dev/null
+	exit 0;
```

Comment:
I think that `;` is not necessary here, as there is only a single command on the line. `;` separates multiple command written on one line.